pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -520,6 +523,17 @@ func ignoreValidatingWebhookMatchConditions(new, old []admissionregistration.Val
 	return true
 }
 
+// ignoreMatchConditions returns true if any new expressions are added
```

Comment:
nit: `ignoreValidatingAdmissionPolicyMatchConditions`?
I understand the comment tried to say that we set opt ignoreMatchConditions to true if any new expressions are added while updating but it might not so clear as the func name doesn't match the comment :). Maybe something like `ignoreValidatingAdmissionPolicyMatchConditions returns true if both paramsKind and matchConditions remain the same while updating hence the update in ValidatingAdmissionPolicySpec will skip matchConditions.` ?

Reply:
In the spirit of making comments semantically valuable and avoiding describing logic, how about:

"ignoreValidatingAdmissionPolicyMatchConditions returns true if there have been no updates that could invalidate previously-valid match conditions"?

Reply:
Sure! 🤗